htlcswitch/switch.go

```diff
@@ -2496,6 +2496,30 @@ func (s *Switch) GetLinksByInterface(hop [33]byte) ([]ChannelUpdateHandler,
 	return handlers, nil
 }
 
+// GetLinksByPubkey fetches all the links connected to a particular node
+// identified by the serialized compressed form of its public key.
+func (s *Switch) GetLinksByPubkey(hop [33]byte) ([]ChannelInfoProvider,
```

Comment:
i think this belongs in the next commit?

Reply:
Moved it to the next one.